src/plugins/base/base.js

```diff
@@ -121,6 +121,16 @@ module.exports = function () {
           marginBottom: '0',
         },
       },
+      // LINK
+      // -----------------------------------------------------------------------
+      'a': {
+        color: config('theme.colors.bright-blue.default'),
+        textDecoration: 'underline',
+        fontWeight: config('theme.fontWeight.semibold'),
+        '&:hover, &:focus, &:active': {
+          color: config('theme.colors.bright-blue.dark'),
+        }
+      },
```

Comment:
I've got some work in progress in the `v7-tw-sm` branch for this as well. Don't merge it in yet as it is not ready.

Reply:
Sure - won't merge until you say it's ok 😃